package.json

```diff
@@ -1,27 +1,34 @@
 {
-  "name": "slynite.com",
-  "version": "2023.09.01",
-  "private": true,
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
-  },
-  "dependencies": {
-    "@heroicons/react": "^2.0.18",
-    "@types/node": "20.3.1",
-    "@types/react": "18.2.13",
-    "@types/react-dom": "18.2.6",
-    "autoprefixer": "10.4.14",
-    "eslint": "8.43.0",
-    "eslint-config-next": "13.4.6",
-    "next": "^13.4.13",
-    "postcss": "8.4.24",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "tailwindcss": "3.3.2",
-    "tailwindcss-animated": "^1.0.1",
-    "typescript": "5.1.3"
-  }
+	"name": "slynite.com",
+	"version": "2023.09.01",
+	"private": true,
+	"scripts": {
+		"dev": "next dev",
+		"build": "next build",
+		"start": "next start",
+		"lint": "next lint",
+		"i18n-to-ts": "ts-node ./src/scripts/convert-langu-json-to-ts-interface.ts"
+	},
+	"dependencies": {
+		"@heroicons/react": "^2.0.18",
+		"@types/node": "20.3.1",
+		"@types/react": "18.2.13",
+		"@types/react-dom": "18.2.6",
+		"autoprefixer": "10.4.14",
+		"eslint": "8.43.0",
+		"eslint-config-next": "13.4.6",
+		"fs": "^0.0.1-security",
+		"json-schema-to-typescript": "^13.0.2",
+		"json-to-interface-generator-for-typescript": "^1.0.1",
+		"json-to-typescript": "^1.0.1",
```

Comment:
Why are you using a dependency which is updated 4 years ago and is not much popular?

Also this only creates interface files where every prop has a `?`. With this behaviour it is not possible to be type safety.